src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const obj of actions) {
```

Comment:
You should take a look at the checklist again
We need to use proper naming in `for...of` loop

Reply:
Done